### PR TITLE
Update teams-meetings-with-canvas.md

### PR DESCRIPTION
**Pull request by mmast-msft**
Updated per feedback from Instructure.  Dev key must be turned on as a first step for the Canvas admin, not the last.

### CODE REVIEW

File: microsoft-365/lti/teams-meetings-with-canvas.md [hunk 1]
```diff
@@ -72,25 +72,32 @@ As a Canvas Admin, you'll need to add the Microsoft Teams meetings LTI app withi
...
+8. Enter the Client ID provided, and then select **Submit**.
 
    You'll notice the Microsoft Teams meetings LTI app name for the Client ID for confirmation.
 
-5. Select **Install**.
+9. Select **Install**.
 
    The Microsoft Teams meetings LTI app will be added to the list of external apps.
 
-6. Enable the app by navigating to the developer keys in the Canvas admin account, selecting inherited, and turning the toggle "on" for Microsoft Teams Meetings.
```
Comment (JohanFreelancer9) [inline microsoft-365/lti/teams-meetings-with-canvas.md]:
```suggestion
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @mmast-msft, kindly commit the suggested change to remove the extra blank line. Many thanks.

**Comment by mmast-msft** [comment]
Committed.  Thanks!

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@DaniEASmith Please merge this PR. Thanks

**Pull request merged**